.travis.yml

```diff
@@ -35,12 +34,12 @@ matrix:
     - rvm: 2.7.1
       gemfile: gemfiles/multi_json.gemfile
       script:
-        - bundle exec rake
+        - bundle exec rubocop
```

Comment:
This line should be removed and default to `rake spec`, we're running all the usual tests, not rubocop.